scenarios/kubernetes_e2e.py

```diff
@@ -363,12 +363,17 @@ def set_up_kops_aws(workspace, args, mode, cluster, runner_args):
     if args.aws_cluster_domain:
         cluster = '%s.%s' % (cluster, args.aws_cluster_domain)
 
+    # AWS requires a username (and it varies per-image)
+    ssh_user = args.kops_ssh_user
```

Comment:
nit: just do `ssh_user = args.kops_ssh_user or 'admin'`?

Reply:
Wow - my python is rusty !  Or should I say go-y :-)

Reply:
or we should ditch the logic here and move to kubetest? :-)